lm75bd/lm75bd.c

```diff
@@ -26,11 +26,25 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
-  return ERR_CODE_SUCCESS;
+    if (temp == NULL) {
+        return ERR_CODE_INVALID_ARG;
+    }
+
+    error_code_t errCode;
+    uint8_t writeBuffer[1];
+    uint8_t readBuffer[2];
```

Comment:
Good practice is to initialize buffers and variables, the errCode above is the exception